### PR TITLE
optimize log in metamanager

### PR DESCRIPTION
**Pull request by Shelley-BaoYue**
Signed-off-by: Shelley-BaoYue <baoyue2@huawei.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**
Message content may contain sensitive information.  Its not suitable to log all message info, just log key fields such as resource and ID.

### CODE REVIEW

File: edge/pkg/metamanager/process.go [hunk 1]
```diff
@@ -293,13 +293,13 @@ func (m *metaManager) processRemoteQuery(message model.Message) {
...
 			time.Duration(metaManagerConfig.Config.RemoteQueryTimeout)*time.Second)
-		klog.Infof("########## process get: req[%+v], resp[%+v], err[%+v]", message, resp, err)
 		if err != nil {
-			klog.Errorf("remote query failed: %v", err)
+			klog.Errorf("remote query failed, req[%s], err: %v", msgDebugInfo(&message), err)
 			feedbackError(err, "Error to query meta in DB", message)
 			return
 		}
 
+		klog.Infof("########## process get: req[%s], resp[%s]", msgDebugInfo(&message), msgDebugInfo(&resp))
```
Comment (wackxu) [inline edge/pkg/metamanager/process.go:302]:
```suggestion
		klog.Infof("remote query get: req[%s], resp[%s]", msgDebugInfo(&message), msgDebugInfo(&resp))
```

Comment (Shelley-BaoYue) [reply edge/pkg/metamanager/process.go]: > 

Done. changed with "process remote query"

**Review by vincentgoat — COMMENTED** [review]
lgtm

File: edge/pkg/metamanager/process.go [hunk 2]
```diff
@@ -293,13 +293,13 @@ func (m *metaManager) processRemoteQuery(message model.Message) {
...
 			time.Duration(metaManagerConfig.Config.RemoteQueryTimeout)*time.Second)
-		klog.Infof("########## process get: req[%+v], resp[%+v], err[%+v]", message, resp, err)
 		if err != nil {
-			klog.Errorf("remote query failed: %v", err)
+			klog.Errorf("remote query failed, req[%s], err: %v", msgDebugInfo(&message), err)
 			feedbackError(err, "Error to query meta in DB", message)
 			return
 		}
 
+		klog.Infof("process remote query: req[%s], resp[%s]", msgDebugInfo(&message), msgDebugInfo(&resp))
```
Comment (fisherxu) [inline edge/pkg/metamanager/process.go:302]: Seems that this log is better for debuging?
```suggestion
		klog.V(4).Infof("process remote query: req[%s], resp[%s]", msgDebugInfo(&message), msgDebugInfo(&resp))
```

Comment (Shelley-BaoYue) [reply edge/pkg/metamanager/process.go]: > Seems that this log is better for debuging?

Done.

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**